Add a text selection handle for the collapsed case.

mpcomplete opened the pull request:
Also simplify handle drawing. All 3 cases are the same, just rotated.

Also fix selection changes on iOS.

Hixie commented on packages/flutter/lib/src/material/input.dart:
```diff
@@ -304,36 +311,20 @@ class _FormFieldData {
 /// points (e.g. the [left] handle points up and to the right).
 class _TextSelectionHandlePainter extends CustomPainter {
-  _TextSelectionHandlePainter({this.type, this.color});
+  _TextSelectionHandlePainter({this.color});
```
spaces inside the braces

mpcomplete replied:
done

Hixie commented:
![LGTM](http://junkyard.damowmow.com/563)

mpcomplete committed:
Add a text selection handle for the collapsed case.

Also simplify handle drawing. All 3 cases are the same, just rotated.

Also fix selection changes on iOS.
